iree/tools/CMakeLists.txt

```diff
@@ -253,13 +254,41 @@ if(${IREE_BUILD_COMPILER})
     PUBLIC
   )
 
+  iree_cc_library(
+    NAME
+      iree_translate_main
+    SRCS
+      "translate_main.cc"
+    DEPS
+      ::init_compiler_modules
+      ::init_iree_passes_and_dialects
+      ::init_mlir_passes_and_dialects
+      ::init_targets
+      ::init_translations
+      ::init_xla_dialects
+      LLVMSupport
+      MLIRIR
+      MLIRSCFTransforms
+      MLIRPass
+      MLIRSupport
+      MLIRTranslation
+      iree::compiler::Conversion::init_conversions
+      iree::compiler::Dialect::VM::Target::Bytecode
+      iree::compiler::Dialect::VM::Target::init_targets
+      iree::compiler::Translation::IREEVM
+    PUBLIC
+  )
+endif(${IREE_BUILD_COMPILER})
+
+if(${IREE_BUILD_COMPILER} OR CMAKE_CROSSCOMPILING)
```

Comment:
Can you add a comment explaining why the `iree_cc_binary` targets are included if `CMAKE_CROSSCOMPILING` is set and `IREE_BUILD_COMPILER` isn't? I'm a little confused - what happens if you try to just compile the runtime for Android? Would that trigger cross compilation and build the compiler components on the host anyway?

Comments here could help.

Reply:
Actually I only need iree-translate here. Reduced the scope and added comment.